File: server/utils/log.go
package utils

import (
	"fmt"
	"server/value"
)

func Ln(msg string) {
	fmt.Println(value.Sslog + value.NOR_ + msg)
}
func Lw(msg string) {
	fmt.Println(value.Sslog + value.WAR_ + msg)
}
func Le(msg string) {
	fmt.Println(value.Sslog + value.ERR_ + msg)
}

File: textual/utils/log.py
import typing

from value.strings import sslog

def ln(msg: typing.Optional[str] = None):
    print(sslog + '[NOR] ' + msg)

def lw(msg: typing.Optional[str] = None):
    print(sslog + '[WAR] ' + msg)

def le(msg: typing.Optional[str] = None):
    print(sslog + '[ERR] ' + msg)
